os/os.proto

```diff
@@ -334,7 +343,7 @@ message StandbyState {
   enum State {
     UNSPECIFIED = 0;
     // The Target does not support dual Supervisors.
-    UNSUPORTED = 1;
+    UNSUPPORTED = 1;
```

Comment:
given the field name changes, should we bump the version of the service as well?

Reply:
Done